Drafting/output

Leafrancia opened the pull request:
Draft of the output part of the project proposal for the YOS award

Leafrancia committed:
Draft proposition for output section

robertomediavilla committed:
edits the "Benefits" section

robertomediavilla commented:
First section ("Benefits") edited. The second sub-section ("Lessons learnt") is not focused. You can keep the introduction in that subsection but then you need to move to list actual lessons learnt, e.g., how hard it is to write a manuscript using GitHub (and why it is important), importance of sharing preprints on ResearchGate for instance, etc.

DaniMori commented on output/Form.md:
```diff
@@ -23,10 +24,39 @@ title: Yerun Open Science Award form
 -   [*Benefits*]{.underline}*: What benefits were realised (to date) through
```
Please @Leafrancia , note there is only one text field (where it says `<!--# blank field -->`) so all the text should go there.

DaniMori commented on output/Form.md:
```diff
@@ -23,10 +24,39 @@ title: Yerun Open Science Award form
 -   [*Benefits*]{.underline}*: What benefits were realised (to date) through
     this initiative, and for whom?*
 
+As a leading international research group and a World Health Organization's
```
I'm not sure if we can call ourselves "leading research group"...

DaniMori commented on output/Form.md:
```diff
@@ -23,10 +24,39 @@ title: Yerun Open Science Award form
+comprehensive way, such as statistical code, manuscript drafts, or meeting
+minutes. By taking this approach, we have not only foster a stronger open
+science practice within our lab, but also across different research consortia in
+the European Union (EU). This may potentially spread through different
```
This sounds great and is very well written. However, I have the concern that this is not true, at least up to now, Remember we are talking here about "what we have achieved so far".

DaniMori commented on output/Form.md:
```diff
@@ -23,10 +24,39 @@ title: Yerun Open Science Award form
+collaborating centre, we work daily with researchers and professionals living
+abroad. This requires tools that help tracking exchanges between labs in a
+comprehensive way, such as statistical code, manuscript drafts, or meeting
+minutes. By taking this approach, we have not only foster a stronger open
```
"not only _fostered_..."

robertomediavilla reviewed (changes requested):
First section ("Benefits") edited. The second sub-section ("Lessons learnt") is not focused. You can keep the introduction in that subsection but then you need to move to list actual lessons learnt, e.g., how hard it is to write a manuscript using GitHub (and why it is important), importance of sharing preprints on ResearchGate for instance, etc.

DaniMori commented on output/Form.md:
```diff
@@ -23,10 +24,39 @@ title: Yerun Open Science Award form
+science practice within our lab, but also across different research consortia in
+the European Union (EU). This may potentially spread through different
+institutes in the region and improve reproducibility rates for the whole
+research community.For example,in the EU-funded project RESPOND, we have already
```
"community. For example" (space after the point)

DaniMori commented on output/Form.md:
```diff
@@ -23,10 +24,39 @@ title: Yerun Open Science Award form
+science practice within our lab, but also across different research consortia in
+the European Union (EU). This may potentially spread through different
+institutes in the region and improve reproducibility rates for the whole
+research community.For example,in the EU-funded project RESPOND, we have already
```
Has that article used Github actually? If you want an example that uses it, you can link this repo: https://github.com/CCOMS-UAM/mapping-initiatives

DaniMori replied:
Also, mind these sentences dismissed from the "Description" section:

https://github.com/CCOMS-UAM/yos-award/blob/29e6f2eff0a4952bc8a02a103b6d237f2c23602b/output/Form.md?plain=1#L65

DaniMori commented on output/Form.md:
```diff
@@ -23,10 +24,39 @@ title: Yerun Open Science Award form
+policies for vulnerable populations during the COVID-19 pandemic, especially
+HCWs. Soon, we will report the RCT results and make the code available. This
+will allow researchers to have full access not only to the results of the study
+but also to how the study developed.
```
Great! This is exactly what I try to convey in "Description"

DaniMori commented on output/Form.md:
```diff
@@ -23,10 +24,39 @@ title: Yerun Open Science Award form
+wars, etc., research should (and will) move rapidly and collaboratively. This
+requires training and collaboration, which is why we apply for this grant. One
+of the main benefits of developing this practice is having a complete history
+accessible to all. This increases the transparency of a study and facilitates
```
Excellent! Mind also these lines dismissed from "Description":

https://github.com/CCOMS-UAM/yos-award/blob/29e6f2eff0a4952bc8a02a103b6d237f2c23602b/output/Form.md?plain=1#L58

DaniMori commented on output/Form.md:
```diff
@@ -23,10 +24,39 @@ title: Yerun Open Science Award form
+accessible to all. This increases the transparency of a study and facilitates
+communication. Moreover, it is an interesting tool for the researcher himself.
+Indeed, having this history allowed us to keep in writing what was done, what
+worked but also what led to an error. This written trace allowed us to save time
```
"what led _to error_"

DaniMori commented on output/Form.md:
```diff
@@ -23,10 +24,39 @@ title: Yerun Open Science Award form
+and avoid reproducing the same mistake while understanding why. Also, when we
+want to remember the previous steps and what has already been done by our peers,
+it is easy to navigate within it once familiar with the platform and find this
+information.
```
This is great! I would also add something about the use of Opal (you may need help/input from @robertomediavilla and/or @Cristrinaranjus for that).

Also, something about the challenges of using these technologies would be in place. That would help us justify the "Follow-up" section (investing in training courses)

DaniMori commented on output/Form.md:
```diff
@@ -23,10 +24,39 @@ title: Yerun Open Science Award form
+it is easy to navigate within it once familiar with the platform and find this
+information.
+
+<!--# blank field, 321 words -->
```
Leave only word count, i.e.:

 ```
<!--# 321 words -->
```

DaniMori reviewed (changes requested):
Well done @Leafrancia ! I really have only two suggestions, the rest are comments we may want to discuss or consider further.

Leafrancia committed:
Arranging the script according to comments

DaniMori committed:
Formats the word count as HTML comment

robertomediavilla committed:
adds section about Opal and DataShield,at Lea's request

robertomediavilla committed:
resolves conflicts

Merge branch 'drafting/output' of github.com:CCOMS-UAM/yos-award into drafting/output

# Conflicts:
#	output/Form.md

robertomediavilla commented:
@DaniMori I think your approval is pending on this pull request. thanks!

DaniMori commented on output/Form.md:
```diff
@@ -57,22 +23,49 @@ title: Yerun Open Science Award form
+work daily with researchers and professionals living abroad. This requires tools
+that help tracking exchanges between labs in a comprehensive way, such as
+statistical code, manuscript drafts, or meeting minutes. By taking this
+approach, we aims not only to develop a stronger open science practice within
```
"we _aim_"

DaniMori commented on output/Form.md:
```diff
@@ -57,22 +23,49 @@ title: Yerun Open Science Award form
+our lab, but also across different research consortia in the European Union
+(EU). We are currently in a test phase of implementing this open science way of
+working in our research group. This may potentially spread through different
+institutes in the region and improve reproducibility rates for the whole
```
"institutions"?

DaniMori commented on output/Form.md:
```diff
@@ -57,22 +23,49 @@ title: Yerun Open Science Award form
+(EU). We are currently in a test phase of implementing this open science way of
+working in our research group. This may potentially spread through different
+institutes in the region and improve reproducibility rates for the whole
+research community. For example,in the EU-funded project RESPOND, we have
```
space after comma

DaniMori commented on output/Form.md:
```diff
@@ -57,22 +23,49 @@ title: Yerun Open Science Award form
+Because a common anonymisation process across datasets was not possible (due to
+the sensitive nature of the data), we used scripts in R, Stata, and SPSS, based
+on each partners' preference, to keep the anonymisation as reproducible as
+possible at all study sites. One of the main benefits of developing this
```
Empashize here that they are all "text-based", and thus susceptible to version control?

DaniMori commented on output/Form.md:
```diff
@@ -57,22 +23,49 @@ title: Yerun Open Science Award form
+possible at all study sites. One of the main benefits of developing this
+practice is having a complete history accessible to all. This increases the
+transparency of a study and facilitates communication. Moreover, it is an
+interesting tool for the researcher himself. Indeed, having this history allowed
```
"researchers themselves" (gender neutral)

DaniMori commented on output/Form.md:
```diff
@@ -57,22 +23,49 @@ title: Yerun Open Science Award form
+interesting tool for the researcher himself. Indeed, having this history allowed
+us to keep in writing what was done, what worked but also what led to errors.
+This written trace allowed us to save time and avoid reproducing the same
+mistake while understanding why. Also, when we want to remember the previous
```
"mistakes"

DaniMori reviewed (approved):
My suggestions are only very minor and can be implemented in the final review of the template.

DaniMori committed:
Merge branch 'drafting/template-fillin' into drafting/output
